fastify.js

```diff
@@ -220,6 +220,7 @@ function build (options) {
       cb = address
       address = undefined
     }
+    address = address || '127.0.0.1'
```

Comment:
I think the PR was correct before this change. This line doesn't allow users to pass `undefined` at all. Sometimes it's necessary to listen on any address in development (i.e. to be able to access the server running on a laptop from a phone).

Reply:
@nwoltman that's when you pass `0.0.0.0` or `::` as the address.

Reply:
Ah true, that's still possible. The [http docs](https://nodejs.org/dist/latest-v9.x/docs/api/net.html#net_server_listen) say:

> If host is omitted, the server will accept connections on the unspecified IPv6 address (`::`) when IPv6 is available, or the unspecified IPv4 address (`0.0.0.0`) otherwise.
> Note: In most operating systems, listening to the unspecified IPv6 address (::) may cause the net.Server to also listen on the unspecified IPv4 address (0.0.0.0).

I'm going to assume `::` is essentially the same as using `undefined`.